README.md

```diff
@@ -1,6 +1,6 @@
 <!-- markdownlint-disable-next-line -->
 <p align="center">
-  <a href="https://mui.com/" rel="noopener" target="_blank"><img width="150" src="https://mui.com/static/logo.svg" alt="MUI logo"></a>
+  <a href="https://mui.com/" rel="noopener" target="_blank"><img width="150" src="/docs/public/static/logo.svg" alt="MUI logo"></a>
```

Comment:
This breaks https://www.npmjs.com/package/@mui/material

<img width="787" alt="Screenshot 2023-02-14 at 18 03 11" src="https://user-images.githubusercontent.com/3165635/218806609-34d3d7c4-e49e-46d9-89a5-9e0a1174cecb.png">